src/views/Popup/routes/Home.tsx

```diff
@@ -31,6 +40,10 @@ import arweaveLogo from "../../../assets/arweave.png";
 import verto_light_logo from "../../../assets/verto_light.png";
 import verto_dark_logo from "../../../assets/verto_dark.png";
 import styles from "../../../styles/views/Popup/home.module.sass";
+import copy from "copy-to-clipboard";
```

Comment:
Just to make sure that everything is organized, let's try to keep the order of imports:
- First imports that don't import the default export of a module (`import { Button }...`)
- Second imports that import the default export of the module (`import copy from...`)
- And the last part should always be the CSS module imports